packages/host/app/components/operator-mode/container.gts

```diff
@@ -283,26 +280,14 @@ export default class OperatorModeContainer extends Component<Signature> {
       let newItem: StackItem = {
         card: chosenCard,
         format: 'isolated',
+        stackIndex: 0, // This is called when there are no cards in the stack left, so we can assume the stackIndex is 0
       };
       this.addToStack(newItem);
     }
   });
 
-  @action
-  isBuried(stackIndex: number) {
-    return stackIndex + 1 < this.stack.length;
-  }
-
-  @action
-  async dismissStackedCardsAbove(stackIndex: number) {
-    for (let i = this.stack.length - 1; i > stackIndex; i--) {
-      let stackItem = this.stack[i];
-      await this.close(stackItem);
-    }
-  }
-
   fetchBackgroundImageURL = trackedFunction(this, async () => {
-    let mostBottomCard = this.stack?.[0]?.card;
+    let mostBottomCard = this.stacks[0]?.items[0]?.card;
```

Comment:
Eventually, we want each stack to have its own background URL

Reply:
Noted. I added a comment to change this later.